scrapyd/utils.py

```diff
@@ -114,7 +114,9 @@ def get_spider_list(project, runner=None, pythonpath=None, version=''):
     if proc.returncode:
         msg = err or out or 'unknown error'
         raise RuntimeError(msg.splitlines()[-1])
```

Comment:
@redapple,
I think we should also decode (from utf8) here
and then, if we are on python2, again encode the msg to unicode_escape before raising.
@kmike often advised this in scrapy.
Shall I commit it directly to master?
I know that the same "FIXME" comment will apply to this line too.